Dockerfile

```diff
@@ -1,6 +1,8 @@
 FROM openjdk:17-jdk-slim
+ARG build_type=
+ENV MODE=${build_type}
```

Comment:
It might make more sense to call this variable PROFILE or ACTIVE_PROFILE since that is what is used as. Not a big deal thought can change it int the future.

Reply:
I feel MODE is more reflective of the expected variable containment. MODE indicates a specification of type, whereas PROFILE to me indicates a user based action.